packages/ckeditor5-paste-from-office/tests/_data/bookmark/image/model.word.html

```diff
@@ -0,0 +1,11 @@
+<paragraph>
+<bookmark bookmarkId="single_image"></bookmark>
+</paragraph>
+<paragraph></paragraph>
+<paragraph>
+<bookmark bookmarkId="multiple_images"></bookmark>
+</paragraph>
+<paragraph>
+<bookmark bookmarkId="image_and_text"></bookmark>Before  after
+</paragraph>
+<paragraph>Foo <bookmark bookmarkId="image_in_text"></bookmark> bar</paragraph>
```

Comment:
This test indicates that there should be some images. I'm not sure if those are not here as bookmarks broke it or what happened.